test/server/data/page/src.html

```diff
@@ -104,6 +104,12 @@
     <use xlink:href="http://domain.com/test.svg#rect"></use>
 </svg>
 <div xlink:href="http://domain.com/test.svg#rect"></div>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink= "http://www.w3.org/1999/xlink">
+    <image xlink:href="https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png">
```

Comment:
Do we need real images in test data?
I guess we can use fake urls for this case.